pulp_rpm/app/depsolving.py

```diff
@@ -838,7 +838,6 @@ def resolve_dependencies(self, unit_repo_map):
             for unit in units:
                 if unit.pulp_type in {"rpm.package", "rpm.modulemd", "rpm.modulemd_defaults"}:
                     solvables.append(self.mapping.get_solvable(unit.pk, repo))
-                else:
                     passthrough[repo].add(unit.pk)
```

Comment:
I think this line needs to be de-indented to break it out of the conditional, otherwise content units like errata won't be "passed through"

Reply:
Oh ugh, concur - let me fix and do more testing